Tests/MiscTests.swift

```diff
@@ -10,9 +10,8 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import UIKit
 import XCTest
-@testable import OktaJWT
+@testable import OktaJWTLib
```

Comment:
I wasn't able to catch this, but why are we renaming the module?

Reply:
It is not renaming the module, it's another module used.
**OktaJWT** - this is the name of test app module, while **OktaJWTLib** - is the name of lib module.
In previous implementation, we included all lib files to both _lib_ and _test app_ modules manually. I've changed this. Made _test app_ and _UTs_ targets dependent on _OktaJWTLib_ module, removed lib files from those targets. Therefore, I've imported the lib module in tests.